apps/examples/wc-next10/package.json

```diff
@@ -0,0 +1,22 @@
+{
+  "name": "wc-next10",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
```

Comment:
To make it more similar to the Next 13 example, I'd add this script for generating a static build:
`"build-static": "next build && next export -o build",`